.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -50,12 +50,12 @@ Push access to the upstream repository is required in order to publish the new t
   - [ ] `cargo publish`
 
 - assemble vendor archive:
-  - [ ] `cargo vendor`
+  - [ ] `cargo vendor target/vendor`
```

Comment:
I'm just curious about this, is it mainly for clarity?

Reply:
`cargo clean` cleans everything in `target/` and I thought we might as well put this in target as well since `target/` is supposed to be the "directory for all generated artifacts and intermediate files". Though it's interesting how the default output path of `cargo vendor` isn't `target/vendor`, so maybe I'm wrong.

Reply:
Ah, now I see, yes I like this idea. Did you manually check that the `tar` changes below produces the same output tarball, i.e. that there isn't an extra `target/` prefix in the archive paths?

Reply:
Yes, I think the output tarball is the same, since it's specified in the `tar` command.

Reply:
Sure, but each of the paths in the archive will now be prefixed by `target/vendor/`.

Reply:
ah, ok, I see what you mean now. Sorry, I was wrong, there is indeed an extra `target/` prefix.

Reply:
I've updated it to use `-C target` so we `tar` changes directory into `target/` first. This way the prefixes of the paths in the archive will just be `vendor`, like before.